tests/integration/test_install_basic.py

```diff
@@ -92,6 +92,23 @@ def test_install_without_dev(pipenv_instance_private_pypi):
 
 @pytest.mark.basic
 @pytest.mark.install
+def test_install_with_version_req(pipenv_instance_pypi):
```

Comment:
I would recommend naming this test like: `test_install_with_version_req_default_operator`

Reply:
Done. Also fixed some failing test. Not sure how to trigger the CI again to check. Test are failing on main branch for me locally, trying to the get working , but right now not 100% I fixed them all